paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetResult.kt

```diff
@@ -33,7 +29,6 @@ sealed class CustomerSheetResult {
      * An error occurred when presenting the sheet
      */
     @ExperimentalCustomerSheetApi
-    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
     class Error internal constructor(
```

Comment:
Just noticing this now, but if the other classes are `Selected` and `Canceled`, should this one be `Failed` for consistency?